src/main/java/in/projecteka/consentmanager/hipLinkInitialization/Constants.java

```diff
@@ -0,0 +1,12 @@
+package in.projecteka.consentmanager.hipLinkInitialization;
+
+import static in.projecteka.consentmanager.common.Constants.CURRENT_VERSION;
+
+public class Constants {
```

Comment:
If you move the package under consentmanager/link, there is already a constant file available there

Reply:
done